.github/workflows/build.sh

```diff
@@ -29,5 +26,17 @@ else
     echo "${ldd_result}" | grep -P "cublas|rocblas" && exit 15
 fi
 
+# Verify that tester linked with intended CPU BLAS.
+echo "${ldd_result}" | grep -P "lib\S*${blas}" || exit 16
```

Comment:
With `bla_vendor`, then `blas` is not set, so this doesn't really check anything.